packages/components/typography/src/Text.tsx

```diff
@@ -44,8 +44,8 @@ function _Text<E extends React.ElementType = typeof DEFAULT_TAG>(
     fontSize = 'fontSizeM',
     fontStack = 'fontStackPrimary',
     fontWeight = 'fontWeightNormal',
-    fontColor = 'gray700',
-    lineHeight,
+    fontColor = 'gray800',
+    lineHeight = 'lineHeightDefault',
```

Comment:
I think we decided to deprecate the tokens that have "functions" in their names like "default"
and we wanted to use the tokens named with their sizes, because it's more scalable

more in this [thread](https://contentful.slack.com/archives/GFB336BAA/p1612430272018100)

and based on this Figma file (Outdated, but the only one I could find... posted the link of the file in slack)
![Screenshot 2021-09-06 at 15 28 31](https://user-images.githubusercontent.com/6597467/132224626-9cd26314-ce1b-4a88-bc1d-e9f1a526ea3b.png)



the line height here is `lineHeightM`

Reply:
got it